src/hera/workflows/v5/task.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+from typing import Dict, List, Optional, Union
+
+from hera.workflows.models import (
+    Arguments,
+    ContinueOn,
+    DAGTask as _ModelDAGTask,
+    Item,
+    LifecycleHook,
+    Sequence,
+    Template,
+    TemplateRef,
+)
+from hera.workflows.v5._mixins import _SubNodeMixin, _TemplateMixin
+from hera.workflows.v5.operator import Operator
+from hera.workflows.v5.task_result import TaskResult
+from hera.workflows.v5.workflow_status import WorkflowStatus
+
+
+class Task(_SubNodeMixin):
+    name: str
+    arguments: Optional[Arguments] = None
+    continue_on: Optional[ContinueOn] = None
+    dependencies: Optional[List[str]] = None
+    depends: Optional[str] = None
+    hooks: Optional[Dict[str, LifecycleHook]]
```

Comment:
Missing a `None`?

Reply:
Copied from the existing task files, will add.